src/__tests__/assets/v1/v1.3.0/multi_send.json

```diff
@@ -3,11 +3,19 @@
   "released": true,
   "contractName": "MultiSend",
   "version": "1.3.0",
-  "codeHash": "0x0208282bd262360d0320862c5ac70f375f5ed3b9d89a83a615b4d398415bdc83",
-  "addresses": {
-    "canonical": "0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761",
-    "eip155": "0x998739BFdAAdde7C933B942a68053933098f9EDa",
-    "zksync": "0x0dFcccB95225ffB03c6FBB2559B530C2B7C8A912"
+  "deployments": {
+    "canonical": {
+      "address": "0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761",
+      "codeHash": "0x0208282bd262360d0320862c5ac70f375f5ed3b9d89a83a615b4d398415bdc83"
+    },
+    "eip155": {
+      "address": "0x998739BFdAAdde7C933B942a68053933098f9EDa",
+      "codeHash": "0x81db0e4afdf5178583537b58c5ad403bd47a4ac7f9bde2442ef3e341d433126a"
```

Comment:
Interesting that the codehash is different here 🤔

Reply:
The `codeHash` for `multiSend` and `SimulateTxAccessor` is different as it uses the `address(this)` within the constructor, thus making a different bytecode based on the address.